compiler/stable_mir/src/abi.rs

```diff
@@ -181,6 +181,7 @@ impl FieldsShape {
 #[derive(Clone, Debug, PartialEq, Eq, Hash, Serialize)]
 pub enum VariantsShape {
     /// Single enum variants, structs/tuples, unions, and all non-ADTs.
+    // FIXME: needs to become `Option` like in the internal type.
```

Comment:
@oli-obk, @celinval, @ouz-a what is the right place to note down "this should be fixed on the next breaking stable MIR update"?

Reply:
I ended up making this a separate variant, and hence also added that variant in stable-mir.